src/compressed_tensors/offload/cache/disk.py

```
@@ -66,13 +66,23 @@ def onload(self, offloaded: torch.Tensor | None) -> torch.Tensor | None:
-            weight_info["safetensors_file"], framework="pt", device=device
-        ) as file:
-            onloaded = file.get_tensor(weight_info["weight_name"])
-            onloaded = to_tensor(onloaded, offloaded)
-            onloaded = onloaded.to(getattr(torch, weight_info["dtype"]))
-            return onloaded
+        try:
+            with safe_open(
+                weight_info["safetensors_file"], framework="pt", device=device
+            ) as file:
+                onloaded = file.get_tensor(weight_info["weight_name"])
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)